sps30-i2c/sps30.h

```diff
@@ -46,6 +46,14 @@ extern "C" {
 #define SPS30_MEASUREMENT_DURATION_USEC 1000000
 /* 50ms delay after resetting the sensor */
 #define SPS30_RESET_DELAY_USEC 50000
+/** SPS30_DEVICE_STATUS_FAN_ERR - The fan is switched on but not running */
+#define SPS30_DEVICE_STATUS_FAN_ERR(device_status) ((device_status) & (1 << 4))
+/** SPS30_DEVICE_STATUS_LASER_ERR - The laser current is out of range */
+#define SPS30_DEVICE_STATUS_LASER_ERR(device_status) \
+    ((device_status) & (1 << 5))
+/** SPS30_DEVICE_STATUS_FAN_SPEED_ERR - The fan speed is out of range */
+#define SPS30_DEVICE_STATUS_FAN_SPEED_ERR(device_status) \
+    ((device_status) & (1 << 21))
```

Comment:
I'm not sure if the macros are that useful for the end user. Maybe it would be better to just have `SPS30_DEVICE_STATUS_FAN_SPEED_ERROR_MASK (1 << 21)` definitions? Then the user can create a mask with for both errors:
```c
uint32_t error_mask = (SPS30_DEVICE_STATUS_FAN_SPEED_ERROR_MASK | SPS30_DEVICE_STATUS_LASER_ERROR_MASK);
```

Reply:
Fixed.